src/connectedk8s/HISTORY.rst

```diff
@@ -3,6 +3,13 @@
 Release History
 ===============
 
+1.1.6
+++++++
+* Moved to track2 SDK
+* `az connectedk8s connect`: Added onboarding timeout parameter
```

Comment:
Should we add _the release namespace thing_ we did few weeks back, here? [PR](https://github.com/Azure/azure-cli-extensions/pull/3611/files)

Reply:
Added